doc/source/extending-jupyterhub.rst

```diff
@@ -81,9 +81,10 @@ configure JupyterHub to build off of this image.
 
    .. code-block:: bash
 
-          numpy=1.12.1
-          scipy=0.19.0
-          matplotlib=2.0
+          pip==9.0.1
```

Comment:
I think we should take this out, since this should be managed by the s2i builder rather than by users (same reason you don't have to specify notebook or ipython here)